rudos.txt

```diff
@@ -12,3 +12,8 @@ Name: Erick Israel Vazquez Neri
 Age: 27
 City: Chihuahua, Chih.
 Hobbies: Music, lecture, code, beer.
+
+Name: Emmanuel Rubio Navarro
+Age: 22
+City: TEpatilán de Morelos, Jalisco
```

Comment:
Has escrito doble uppercase. Pero todo cool :V 👍

Reply:
Fix this.